modules/scope3RtdProvider.md

```diff
@@ -0,0 +1,358 @@
+Apply AEE signals only to certain bidders:
+```javascript
+params: {
+  orgId: 'YOUR_ORG_ID',
+  bidders: ['rubicon', 'appnexus', 'amazon'],
+  advertiserTargeting: true,
+  publisherTargeting: true
+}
+```
+
+#### Development/Testing
+```javascript
+params: {
+  orgId: 'YOUR_ORG_ID',
+  endpoint: 'https://staging.prebid.scope3.com/prebid',
+  timeout: 2000,
+  debugMode: true,
+  cacheEnabled: false
+}
+```
+
+## Data Flow
+
+### 1. Complete OpenRTB Capture
+The module captures ALL available OpenRTB data:
+- **Site**: page URL, domain, referrer, keywords, content, categories
+- **Device**: user agent, geo location, IP, device type, screen size
+- **User**: ID, buyer UIDs, year of birth, gender, keywords, data segments, **all extended IDs (eids)**
+- **Impressions**: ad unit details, media types, floor prices, custom data
+- **Regulations**: GDPR, COPPA, consent strings
+- **App**: if in-app, all app details
+
+### 2. Request to AEE
+Sends the complete OpenRTB request with list of bidders:
+```json
+{
+  "orgId": "YOUR_ORG_ID",
+  "ortb2": {
+    "site": { 
+      "page": "https://example.com/page",
+      "domain": "example.com",
+      "cat": ["IAB1-1"],
+      "keywords": "news,sports"
+    },
+    "device": { 
+      "ua": "Mozilla/5.0...",
+      "geo": {
+        "country": "USA",
+        "region": "CA",
+        "city": "San Francisco"
+      },
+      "ip": "192.0.2.1"
+    },
+    "user": {
+      "id": "user123",
+      "eids": [
+        {
+          "source": "liveramp.com",
+          "uids": [{"id": "XY123456"}]
+        },
+        {
+          "source": "id5-sync.com", 
+          "uids": [{"id": "ID5*abc"}]
+        }
+      ],
+      "data": [...]
+    },
+    "imp": [...],
+    "regs": { "gdpr": 1, "us_privacy": "1YNN" }
+  },
+  "bidders": ["rubicon", "appnexus", "amazon", "pubmatic"],
+  "timestamp": 1234567890,
+  "source": "prebid-rtd"
+}
+```
+
+### 3. AEE Response
+Receives targeting signals with bidder-specific segments and deals:
+```json
+{
+  "aee_signals": {
+    "include": ["sports_fan", "auto_intender", "premium_user"],
+    "exclude": ["competitor_exposed", "frequency_cap_reached"],
+    "macro": "eyJjb250ZXh0IjogImhpZ2hfdmFsdWUiLCAic2NvcmUiOiAwLjg1fQ==",
+    "bidders": {
+      "rubicon": {
+        "segments": ["rub_luxury_auto", "rub_sports_fan"],
+        "deals": ["RUBICON_DEAL_123", "RUBICON_DEAL_456"]
+      },
+      "appnexus": {
+        "segments": ["apn_high_value", "apn_auto_intent"],
+        "deals": ["APX_PREMIUM_DEAL"]
+      },
+      "amazon": {
+        "segments": ["amz_prime_member"],
+        "deals": []
+      }
+    }
+  }
+}
+```
+
+### 4. Signal Application
+
+#### Publisher Targeting (GAM)
+Sets the configured targeting keys:
+- `s3i` (or your includeKey): ["sports_fan", "auto_intender", "premium_user"]
+- `s3x` (or your excludeKey): ["competitor_exposed", "frequency_cap_reached"]
+- `s3m` (or your macroKey): "eyJjb250ZXh0IjogImhpZ2hfdmFsdWUiLCAic2NvcmUiOiAwLjg1fQ=="
```

Comment:
this example includes disallowed gam characters https://support.google.com/admanager/answer/10020177?hl=en; gam will also convert everything to lower case